includes/fs-plugin-info-dialog.php

```diff
@@ -1450,7 +1507,7 @@ class="fs-annual-discount"><?php printf(
             echo "</div>\n";
             echo "</div>\n";
             echo "</div>\n"; // #plugin-information-scrollable
-            echo "<div id='$tab-footer'>\n";
+            echo "<div id='$_tab-footer'>\n";

```

Comment:
@fajardoleo better to wrap the var with `{}`

Reply:
Updated